Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM centos:centos7
 
 RUN yum update -y \
     && yum group install -y "Development Tools" \
-    && yum install -y https://www.klayout.org/downloads/CentOS_7/klayout-0.27-0.x86_64.rpm \
+    && yum install -y https://www.klayout.org/downloads/CentOS_7/klayout-0.27.1-0.x86_64.rpm \
```

Comment:
I guess the CentOS_7 works on Ubuntu but I do see he releases binaries for both if it matters.

Reply:
Currently we only maintain a Dockerfile for centos, not ubuntu.